client/src/partials/accounts/accounts.js

```diff
@@ -28,13 +28,14 @@ function AccountsController($rootScope, $timeout, AccountGrid, Notify, Constants
 
   // lang parameter for document
   vm.parameter = { lang: Language.key };
-  vm.loading = true;
 
   vm.Accounts = new AccountGrid();
   vm.Accounts.settup()
     .then(bindGridData)
     .catch(Notify.handleError)
-    .finally(toggleLoadingIndicator);
+    .finally(function () {
+      toggleLoadingIndicator();
+    });
```

Comment:
They are logically equivalent.  I agree as well. :+1: I was trying to debug something and never switched it back.  Good catch :+1: